Circle_Sort/README.md

```diff
@@ -35,39 +35,39 @@ Let's arrange the numbers in ascending order using circle sort.
 3.Split the array in two parts and recurse until there is only one single element in the array.
 
 **Pseudocode**
-
-> function boolean circleSort(a: array of items, int low, int high)<br />
->  
->   boolean swap = false <br />
->   
->   int l = low, h = high<br />
->   
->   while(l<h)<br />
->     if a[l] > a[h]<br />
->       swap ( a[l], a[h] )<br />
->       swap = true<br />
->       <br />
->    end if<br />
->    
->    l++<br />
->    h--<br />
->    
->   end while<br />
->   
->   if l == hi<br />
->     if( a[l] > a[h+1] )<br />
->       swap ( a[l], a[h+1] )<br />
->       swap=true<br />
->     end if <br />
->     
->   int mid = (high-low)/2<br />
->   bool firstArray=circleSort(a, low, low+mid)<br />
->   bool secondArray=circleSort(a, low+mid+1, high)<br />
->   
->   return swap || firstArray || secondArray<br />
->   
->  end function
->  
+```
+ function boolean circleSort(a: array of items, int low, int high)<br />
```

Comment:
Fix indentation in the code.
Rest looks good.

Reply:
@SakshiShreya  i'm sorry,i didn't get it.
Did you mean to keep space from the left margin?

Reply:
Yes. That is what indentation means.

Reply:
@SakshiShreya Done.